doc/nep1.rst

```diff
@@ -247,6 +247,10 @@ Coding Conventions
   variables remain immutable, and gives those who read the code a better idea
   of the code's purpose.
 
+- Do not use `float` for money.
+
+- Do not use `Natural` or `Positive` as return type, use `int` or `uint` as return type instead.
```

Comment:
araq gave the rationale for that somewhere, please find it and add it so that this rule doesn't appear arbitrary